### ISSUE TITLE
Webview+ not debuggable using 2.4.2

### ISSUE THREAD

**Issue by linasale** [ludei/webview-plus#35]
The webview is not debuggable in Chrome anymore. The previous version worked fine using the Chrome dev tool "Inspect devices". But now the list of webviews is empty.

**Comment by almost** [comment]
Any solutions to this? Does anyone know which versions *do* work?

**Comment by paishin** [comment]
Same problem on 2.4.3

**Comment by almost** [comment]
2.4.0 works with remote chrome debugging.

I was hoping to track down the issues but I discovered that this repository doesn't actually contain any source code. It just contains .JAR files. Not entirely sure what's going on with this?

**Comment by extart** [comment]
The problem is confirmed in 2.4.3.
The source code is not in the repository and the developer does not respond to the problem.

**Comment by paishin** [comment]
No activity in Webview+ for quite some time now, I do not thing is actively maintained any more. Maybe its better to switch to Crosswalk for cordova apps.

**Comment by extart** [comment]
Indeed, the use crosswalk may be a good idea, the source code is open. Here with the correct installation of the plugin can not find out for several months. And to use the cloud compiler quite dangerous, with this approach to the case. Now he works, and tomorrow - all went on vacation to relax. :-)
